zerver/lib/webhooks/git.py

```diff
@@ -37,15 +37,22 @@
 
 def get_push_commits_event_message(user_name, compare_url, branch_name, commits_data, is_truncated=False):
     # type: (Text, Optional[Text], Text, List[Dict[str, Any]], Optional[bool]) -> Text
+    committers = get_all_committers(commits_data)
+    committers_detail = "Commits by"
+    for committer, no_of_commits in committers.items():
+        committers_detail = "{} {}({})".format(committers_detail, committer, no_of_commits)
     if compare_url:
-        pushed_text_message = PUSH_PUSHED_TEXT_WITH_URL.format(compare_url=compare_url)
+        pushed_text_message = PUSH_PUSHED_TEXT_WITH_URL.format(compare_url=compare_url, no_of_commits=len(commits_data),
+                                                               commit_or_commits=commit_or_commits(len(commits_data)))
```

Comment:
you can define this value above:
`commit{}`.format('s' if len(commits_data)>1 else '')`
and use in 2 places